Gemfile

```diff
@@ -2,8 +2,8 @@ source 'https://rubygems.org'
 ruby '2.2.2'
 
 gem "csvlint"
-gem "puppet-lint"
+gem "puppet-lint", "2"
```

Comment:
Our last successful build, and [0.10 build](https://travis-ci.org/coala/coala-bears/jobs/205686455) is 
`Using puppet-lint 2.1.1`
your build is
`Installing puppet-lint 2.0.0`

So this would be a regression.
Please upgrade the `GemRequirement` to 2.1.1